other-docs/guides/migrating-from-wordpress.md

```diff
@@ -41,7 +41,12 @@ Installing Altis will replace your `wp-config.php` so you should back it up if y
 
 ## Installing Altis
 
-Now you have your composer project up and running, it’s time to add Altis to the project. Do so by running `composer require altis/altis`.
+Now you have your composer project up and running, it’s time to add Altis to the project. Do so by following these steps:
+
+1. Remove existing `vendor` directory and `composer.lock` if present
```

Comment:
Hmm I'm not sure this should be the recommended approach. The original instruction was correct, but I guess ran the risk of conflicting packages. I think it would be better to add it as a note "If you get this error, do this", as this really isn't a get elegant recommended way to do it, because we're nuking a project's known-good-versions

Reply:
Ahhh I hit that same error with a specific version of PHPUnit - we're including a known supported version now though so that should help. Will move this to a trouble shooting section 👍

Reply:
Updated